src/main/java/com/example/services/TicketService.java

```diff
@@ -0,0 +1,65 @@
+package com.example.services;
+
+import com.example.models.Ticket;
+import com.example.repository.TicketRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+
+@Service
+public class TicketService {
+
+    private TicketRepository ticketRepository;
+
+    @Autowired
+    public TicketService(TicketRepository ticketRepository){
+        this.ticketRepository = ticketRepository;
+    }
+
+    public TicketService() {
+
+    }
```

Comment:
if you do not need at least 1 case to use instance of TicketService without injected TicketRepository bean - remove this constructor